src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
The reason I can't answer that question confidently is also the only reason I would inherit flags, and that is quite simply because I just don't know. This was probably my state of mind when I added that line.

However, my preference is to know and set every flag deliberately. Honestly, I believe it is the most portable solution. Otherwise we are opening ourselves to all sorts of unreproducible, system-specific bugs.